samples/highcharts/data-tools/mathmodifier-formula/demo.details

```diff
@@ -0,0 +1,7 @@
+---
+ name: MathModifier Formula demo
+ authors:
+   - Sophie Bremer
+ js_wrap: b
+ skipTest: true
```

Comment:
Should we declare skipTest here?

Reply:
`data-tools` is not an official module. This is for internal use to make you understand the data layer.